environments/production/vars.yml

```diff
@@ -21,6 +21,8 @@ datadog:
   site: us5.datadoghq.com
 
 project_timezone: 'US/Pacific'
+enable_postgresql_backup: true
```

Comment:
This was set manually on prod, but persisting it here for consistency. 

(somewhat related, I've noticed that both staging and prod has some uncommited changes, so created [this ticket](https://dimagi.atlassian.net/browse/SC-4248) to rectify that)